growth/too/templates/plan_new.html

```diff
@@ -65,6 +65,10 @@
                         {{ render_field(form.primary, class='custom-control-input') }}
                         <label for=primary class=custom-control-label>Use primary only</label>
                     </div>
+                    <div class="custom-control custom-switch">
+                        {{ render_field(form.balance, class='custom-control-input') }}
+                        <label for=balance class=custom-control-label>Filter balancing (Require field observations in all requested filters)</label>
```

Comment:
Would you please show a screen shot of this? I suspect that if formatted as normal text, it might wrap oddly.

Reply:
Please use https://getbootstrap.com/docs/4.3/components/forms/#help-text.